hack/check-commits.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o pipefail
+set -o nounset
+
+__dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+parent_branch="master"
+git remote add openshift https://github.com/openshift/assisted-installer.git &> /dev/null || true
+git fetch openshift ${parent_branch}
+
+current_branch="$(git rev-parse --abbrev-ref HEAD)"
+
+revs=$(git rev-list "FETCH_HEAD..${current_branch}")
+
+for commit in ${revs};
+do
+    commit_message=$(git cat-file commit "${commit}" | sed '1,/^$/d')
+    tmp_commit_file="$(mktemp)"
+    echo "${commit_message}" > "${tmp_commit_file}"
+    "${__dir}/check-commit-message.sh" "${tmp_commit_file}"
```

Comment:
This works. You can also consider that if you don't want to deal with temporary files, you can always use variable redirection and process substitution
```suggestion
    "${__dir}/check-commit-message.sh" <(echo "$commit_message")
```

It can also be written as:
```
    "${__dir}/check-commit-message.sh" /dev/fd/64 64<<<"$commit_message"
```